ui/pagination-dots/src/play.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { PaginationDots as Component } from "./";
+import { theme, styled } from "@washingtonpost/wpds-theme";
+
+export default {
+  title: "PaginationDots",
+  component: Component,
+};
+
+const DefaultArgs = {
+  amount: 6,
+  index: 1,
+  label: "Progress Bar",
+};
+
+const Container = styled("div", {
+  textAlign: "center",
+});
+const Label = styled("h3", {
+  textAlign: "center",
+  [".wpds-dark &"]: { color: theme.colors.primary },
```

Comment:
```suggestion
  [`${darkTheme} &`]: { color: theme.colors.primary },
```

https://stitches.dev/docs/styling#theme-specific-styles